cephfs/admin/pin_test.go

```diff
@@ -0,0 +1,86 @@
+//go:build !nautilus && ceph_preview
+// +build !nautilus,ceph_preview
+
+package admin
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPinSubVolume(t *testing.T) {
+	fsa := getFSAdmin(t)
+	volume := "cephfs"
+	type svgn struct {
+		group string
+		name  string
+	}
+	created := []svgn{}
+	defer func() {
+		for _, c := range created {
+			err := fsa.RemoveSubVolume(volume, c.group, c.name)
+			assert.NoError(t, err)
+			delay()
+			if c.group != NoGroup {
+				err := fsa.RemoveSubVolumeGroup(volume, c.group)
+				assert.NoError(t, err)
+			}
+		}
+	}()
```

Comment:
I just noticed this when starting to look at your more recent PR...

This type, slice and defer function for cleanup is fine, but possibly a bit over-engineered since you only end up creating one subvolume in this test.
I am happy to let it stand as-is but I will give you the opportunity to simplify it if you want to.  Just let me know what you prefer to do.

Reply:
It's purely copy/paste. But I am in agreement here and we can simply it. I'll update soon.